networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/agent.py

```diff
@@ -133,10 +135,18 @@ def delete_log_precommit(self, context, log_obj):
 
     def resource_update(self, context, log_obj):
         pass
-    
+
     def address_group_updated(self, context, address_group):
         self.callback(address_group, self.realizer.address_group_update)
 
+    def get_realization_status(self, context, port_id):
```

Comment:
```suggestion
    def is_port_realized(self, context, port_id):
```

otherwise I'd kind of expect something else than a boolean to be returned (same for the callers)